### PR TITLE
Publish 1.9.2

### PR DESCRIPTION
**Pull request by jacob-delgado**
Publish 1.9.2

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by jacob-delgado** [comment]
Waiting for qualification tests pass confirmation before publishing.

**Comment by ericvn** [comment]
@jacob-delgado Are there really no changes?

**Comment by craigbox** [comment]
That's a short list of changes

**Comment by craigbox** [comment]
snap

**Comment by craigbox** [comment]
(He just added them)

File: content/en/news/releases/1.9.x/announcing-1.9.2/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - /news/announcing-1.9.2
+---
+
+This release note describes what’s different between Istio 1.9.1 and Istio 1.9.2.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** transport socket parameters are now taken into account when configured in `EnvoyFilter`
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.2/index.md:18]: Grammer nit. 

```suggestion
- **Fixed** an issue so transport socket parameters are now taken into account when configured in `EnvoyFilter`
```

File: content/en/news/releases/1.9.x/announcing-1.9.2/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- **Fixed** a bug causing runaway logs in `istiod` after disabling the default ingress controller.
+  ([Issue #31336](https://github.com/istio/istio/issues/31336))
+
+- **Fixed** the Kubernetes API server is now considered to be cluster-local by default. This means that any
+pod attempting to reach `kubernetes.default.svc` will always be directed to the in-cluster server.
+  ([Issue #31340](https://github.com/istio/istio/issues/31340))
+
+- **Fixed** issue with metadata handling for Azure platform. Support added for
+`tagsList` serialization of tags on instance metadata.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.2/index.md:29]: Grammer nit. It also seems like an issue was fixed and also support was added. Maybe need to combine the two sentences.

```suggestion
- **Fixed** an issue with metadata handling for the Azure platform, allowing
`tagsList` serialization of tags on instance metadata.
```

File: content/en/news/releases/1.9.x/announcing-1.9.2/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Changes
+
+- **Fixed** transport socket parameters are now taken into account when configured in `EnvoyFilter`
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Fixed** a bug causing runaway logs in `istiod` after disabling the default ingress controller.
+  ([Issue #31336](https://github.com/istio/istio/issues/31336))
+
+- **Fixed** the Kubernetes API server is now considered to be cluster-local by default. This means that any
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.2/index.md:24]: Grammer nit:

```suggestion
- **Fixed** an issue so the Kubernetes API server is now considered to be cluster-local by default. This means that any
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**